wire-ios-sync-engine/Source/Synchronization/EventProcessor.swift

```diff
@@ -91,14 +91,39 @@ actor EventProcessor: UpdateEventProcessor {
         bufferedEvents.append(contentsOf: events)
     }
 
+    /// Decrypt Store and Process events from webSocket
+    func processLiveEvents(_ events: [ZMUpdateEvent]) async throws {
+        try await processEvents(events, duringQuickSync: false)
+    }
+
+    /// Decrypt Store and Process events during quickSync
     func processEvents(_ events: [ZMUpdateEvent]) async throws {
+        try await processEvents(events, duringQuickSync: true)
+    }
+
+    private func processEvents(_ events: [ZMUpdateEvent], duringQuickSync: Bool) async throws {
+
         try await enqueueTask {
             NotificationCenter.default.post(name: .eventProcessorDidStartProcessingEventsNotification, object: self)
 
             guard !DeveloperFlag.ignoreIncomingEvents.isOn else { return }
 
             let publicKeys = try? await self.earService.fetchPublicKeys()
+
+            if duringQuickSync {
+                NotificationCenter.default.post(
```

Comment:
question: I guess for the new sync, we just need to post these notifications as well?

Reply:
yes

Reply:
did you create a jira ticket?

Reply:
[WPB-16348](https://wearezeta.atlassian.net/browse/WPB-16348)